doc/examples/rendering/traceSetsArnold.gfr

```diff
@@ -0,0 +1,348 @@
+__children["RenderSetup"]["BoxIn"]["__in"].setInput( __children["RenderSetup"]["in"] )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'nodule:color', imath.Color3f( 0.240099996, 0.339399993, 0.485000014 ) )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'labelPlugValueWidget:renameable', True )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'nodule:type', 'GafferUI::StandardNodule' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'description', 'The input scene' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'plugValueWidget:type', '' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'noduleLayout:spacing', 2.0 )
+__children["RenderSetup"]["__uiPosition"].setValue( imath.V2f( 8.73167801, -126.091125 ) )
+__children["Backdrop"]["title"].setValue( 'Example: Trace Sets (Arnold)' )
+__children["Backdrop"]["description"].setValue( "In this graph we show how to make use of Arnold's trace sets.\n\nArnold allows both objects and rays to be tagged with one or more trace sets. This allows the scene to be compartmentalized, such that affected shading calculations simply dont 'see' objects in other trace sets.\n\nIn this example we will create a mirror that only reflects some of the objects in the scene.\n\n - An object's trace set membership is tagged using Gaffer sets.\n\n - Rays cast from a shader are tagged by inserting a aiTraceSet shader into the shading network before assignment." )
+__children["Backdrop"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, 14.9502449 ), imath.V2f( 30, 46.6711006 ) ) )
+__children["Backdrop"]["__uiPosition"].setValue( imath.V2f( -1.96337986, -19.7618332 ) )
+__children["Backdrop1"]["title"].setValue( 'Adding objects to a trace set' )
+__children["Backdrop1"]["description"].setValue( 'Gaffer manages trace sets using the same nodes as normal sets. \n\nThe only difference is that the `render:` prefix is used to denote that it should be converted to a render trace set.\n\nThe SetSpheres node adds the two spheres to their own trace set using the set name `render:spheres`.' )
+__children["Backdrop1"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, -7.14684677 ), imath.V2f( 30, 13.4110298 ) ) )
+__children["Backdrop1"]["__uiPosition"].setValue( imath.V2f( -34.6775818, -40.1869965 ) )
+__children["Backdrop2"]["title"].setValue( 'Ray tagging' )
+__children["Backdrop2"]["description"].setValue( 'As the spheres are in a custom trace set, we can create a reflective surface that uses this set to control what is visible.\n\nThe aiTraceSet Node inserted after the shading network, before it is assigned modifies the rays traced by the attached network such that they either include or exclude objects in the named trace sets.\n\nTry toggling the Inclusive checkbox to switch the mirror between showing only the spheres or exluding them.\n\n' )
```

Comment:
### Backdrop text
~~As~~ **Now that** the spheres are in a custom trace set, we can create a reflective surface that ~~uses~~ **refers to** this set to control ~~what is~~ **which objects are** visible **in its reflection**.

The ~~aiTraceSet Node inserted~~ **TraceSet node (Arnold > Shader > Utility > Trace Set) is connected** after the shading network, before it is assigned modifies the rays traced by the attached network **[Can you reword this? I can't make sense of it.]** such that they either include or exclude objects in the named trace sets.

~~Try toggling the Inclusive checkbox to switch the mirror between showing only the spheres or exluding them.~~
**With the IPR turned on, try checking the TraceSet node's Inclusive plug on and off. This will toggle between the mirror showing only the spheres, and excluding them.**

Reply:
Theres a bit of a gotcha with the node name/shader name situation. If you are a 'navigate the menu' person, then it shows up under the path you described. If however, you type in `trace` into the search box, then it shows up as `aiTraceSet`.  We probably need to fix the search menu so it shows the actual item text, and suitably disambiguates nodes with the same name (eg: Appleseed/Arnold's StandardSurface).

See what you think to:

> As the spheres are in a custom trace set, we can create a reflective surface that refers to this set to control which objects are visible in its reflection.
>
>The TraceSet node (Arnold > Shader > Utility > Trace Set) is connected after the shading network but before it is assigned. It modifies the rays traced by the network such that they either include or exclude the objects in the specified trace sets.
>
> With the IPR turned on, try checking the Inclusive plug on and off. This will toggle the mirror between showing only the spheres and excluding them.